src/rgw/rgw_lc_s3.cc

```diff
@@ -139,9 +139,20 @@ void LCFilter_S3::decode_xml(XMLObj *obj)
     single_cond = true;
   }
 
-  RGWXMLDecoder::decode_xml("Prefix", prefix, o);
-  if (!prefix.empty())
+  auto prefix_iter = o->find("Prefix");
+  std::string temp_prefix;
+  std::vector<std::string> prefixs;
+  while (auto prefix_xml = prefix_iter.get_next()){
+    decode_xml_obj(temp_prefix, prefix_xml);
```

Comment:
Decoding and storing all the prefixes is overkill here, if we're just going to error out.  Just do a single get_next() and decode, then do another get_next(), and if it returns something, throw the error.

Reply:
thanks, I reset and re-push a new commit.